src/map/utils/charutils.cpp

```diff
@@ -1023,7 +1023,7 @@ namespace charutils
         if (rset)
         {
             // equipSlotData[equipSlotId] = { slotId, containerId }
-            std::unordered_map<uint8, std::pair<uint8, uint8>> equipSlotData;
+            std::map<uint8, std::pair<uint8, uint8>> equipSlotData;
```

Comment:
As part of this, do we need a `SORT ASC/DESC` on the query? Or is there some other mechanism enforcing that?

Reply:
We shouldn't need to, elements of a map are sorted by their key. From what I can gather from the code, and what I've seen in my own testing, the sorting here seems to be based off of SLOTTYPE in battleentity.h.